run.sh

```diff
@@ -346,6 +349,23 @@ check_plugins() {
   done <<< "$(WP plugin list --field=name)"
 }
 
+check_packages() {
+  if [[ ! "${REQUIRE-}" ]]; then
+    h3 "No package dependencies listed"
+    STATUS SKIP
+    return
+  fi
+
+  h3 "Installing composer packages"
+  touch /app/composer.json
```

Comment:
I think that if someone is advanced enough to know how to use composer, they should be able to follow a simple instruction to volume the `composer.json` into the root of the `/app` directory. 

We're already at a point of having way too many environment variable options. Makes things chaotic to keep adding more and more IMHO.

Deferring the judgement call for this to @karellm.